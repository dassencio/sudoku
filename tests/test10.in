3 4
 _ 11 12  1  _  9 10  _  _  _  _  _
10  _  _  _  2  _  _  _  7  6  _  _
 3  6  _  _  _  7  8  _  _  _ 10  _
 _  _  1  5  _  8  _  _  3  _  7  _
 _  _  _  _  _  _  _  _ 12  _  _  _
 _  _  7 12 10  _  1  2  6 11  5  _
 _ 10  8  7  1 12  _  3 11  4  _  _
 _  _  _  4  _  _  _  _  _  _  _  _
 _  9  _  6  _  _ 11  _  1  7  _  _
 _  8  _  _  _  2  5  _  _  _  1  7
 _  _  9 10  _  _  _  8  _  _  _  3
 _  _  _  _  _  3 12  _  4  9  8  _
