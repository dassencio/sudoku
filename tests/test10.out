 7 11 12  1  5  9 10  6  8  3  2  4
10  5  4  8  2 11  3  1  7  6  9 12
 3  6  2  9 12  7  8  4  5  1 10 11
 9  4  1  5 11  8  6 12  3  2  7 10
 6  2 10 11  3  5  7  9 12  8  4  1
 8  3  7 12 10  4  1  2  6 11  5  9
 5 10  8  7  1 12  9  3 11  4  6  2
 1 12 11  4  8  6  2  7  9 10  3  5
 2  9  3  6  4 10 11  5  1  7 12  8
 4  8  6  3  9  2  5 11 10 12  1  7
12  7  9 10  6  1  4  8  2  5 11  3
11  1  5  2  7  3 12 10  4  9  8  6
